src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -227,10 +227,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
     // Insert an execution row
       executionInsert <- dataAccess.executionsAutoInc +=
         new Execution(
-          workflowExecution.workflowExecutionId.get,
-          key.scope.fullyQualifiedName,
-          key.index.fromIndex,
-          ExecutionStatus.NotStarted.toString)
+          workflowExecutionId = workflowExecution.workflowExecutionId.get,
+          callFqn = key.scope.fullyQualifiedName,
+          index = key.index.fromIndex,
+          status = ExecutionStatus.NotStarted.toString,
+          rc = None,
+          startDt = new Date().toTimestamp,
```

Comment:
Maybe the same way you set the end date in setStatus when we reach a terminal state, but for start date when we get to Starting or Running.